app/containers/WriteReviewPage/constants.js

```diff
@@ -50,4 +50,9 @@ export const SELECT_GUIDE_RATING_ACTION = 'app/WriteReviewPage/SELECT_GUIDE_RATI
 // Text input
 export const UPDATE_TRIP_REPORT_DETAILS_ACTION = 'app/WriteReviewPage/UPDATE_TRIP_REPORT_DETAILS_ACTION';
 
+export const CUSTOM_VALIDATION_FAILED_ACTION = 'app/WriteReviewPage/CUSTOM_VALIDATION_FAILED_ACTION';
+
 export const DEFAULT_ACTION = 'app/WriteReviewPage/DEFAULT_ACTION';
+
+export const CUSTOM_VALIDATION_STYLE = { border: '1px solid #dc3545' }; // TODO- get this color from bootstrap somehow
```

Comment:
Missed commenting on this before. I also see this is the constant being used in WriteReviewPage's index.js. Having a constant for styling seems very strange to me. What's the intention here?

Also note, you cannot get custom colors out of Bootstrap. The only ways to do that (without doing this) are through our styled-components library or creating a custom stylesheet. But I saw this is a very red color, and if you want a red border you can add the string 'border border-danger' to the class string of an element.

Reply:
Hah Nice dude!!! Good call. I'll look at fixing this now.

There are 2 places they apply this, it;'s pretty hacky but it's how I'm doing validation on the react-select stuff.